### PR TITLE
Fix framerate option

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: lib/server.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ module.exports = (directory, format, width, height, framerate, horizontalFlip, v
 
   // Set up camera stream conversion
   let conversionStream = ffmpeg(cameraStream.stdout)
+    .inputOptions("-framerate", framerate)
```
Comment (jaredpetersen) [inline lib/server.js:31]: To my knowledge, FFmpeg automatically determines framerate from the input.

Comment (ghost) [reply lib/server.js:31]: I believe it only does so with certain video formats. In my experience with raspi-live, the framerate will default to 25fps without this change.

Comment (andrejpk) [reply lib/server.js:31]: I'm seeing the same issue.. without this line, I'm stuck at 25fps. With this line in, it works.
Any blockers from merging this?

**Review by andrejpk — APPROVED** [review]

**Comment by jaredpetersen** [comment]
@andrejpk I was mainly waiting on someone else to verify it since I don't have a raspberry pi available to test this change. If you were able to verify it as well, it seems reasonable to merge this in.

The only thing remaining is that the library needs a patch revision. I'll try to see if I can modify this PR when I have time since the original author is probably MIA due to how long it's been since they submitted this for review.

**Comment by jaredpetersen** [comment]
Looks like I can't modify the PR and the author has deleted their GitHub account.

@andrejpk If you can submit another PR with this change (and the version patch revision) I'd be happy to merge it in. If not, I will get to it as soon as I can.

**Comment by jaredpetersen** [comment]
Resolved in #29. Closing this out.

**Pull request closed without merging**